source/css/scss/components/_header.scss

```diff
@@ -31,9 +31,62 @@
   width: 1.5em !important;
   height: 1.5em;
   border-radius: 50%;
-  bottom: 4px;
 }
 
+// Hidden, but actually not there
+
+.no-mobile {
+  display: block;
+  visibility: visible;
+
+  &.hide {
+    display: block;
+  }
+
+  &.c-primary-nav__link {
+    &.hide {
+      display: block;
+    }
+    display: flex;
+  }
+
+  @media screen and (max-width: 768px) {
+    &.c-primary-nav__link {
+      display: none;
+
+      &.show {
+        display: none;
+      }
+    }
+    display: none;
+    visibility: hidden;
+  }
+}
+
+.only-mobile, .only-mobile.c-primary-nav__link {
+  display: none;
+  visibility: hidden;
+
+
+  @media screen and (min-width: 768px) {
+    &.show {
+      display: none !important;
```

Comment:
is the !important necessary? hate to see it

Reply:
This should be gone in the newest updates